compiler/luci-interpreter/src/kernels/LogSoftmax.test.cpp

```diff
@@ -78,6 +78,34 @@ TEST(LogSoftmaxTest, Uint8)
               ::testing::ElementsAreArray({189, 93, 221, 253, 142, 63, 255, 111}));
 }
 
+TEST(LogSoftmaxTest, IvalidInputOutputType_NEG)
```

Comment:
`Ivalid` -> `Invalid` ? (same comment for another test)

Reply:
Oh, I will fix it.